client/src/Pos/Client/Txp/Util.hs

```diff
@@ -145,14 +152,14 @@ isCheckedTxError = \case
 
 -- | Specifies the way Utxos are going to be grouped.
 data InputSelectionPolicy
-    = OptimizeForSecurity  -- ^ Spend everything from the address
-    | OptimizeForSize      -- ^ No grouping
+    = OptimizeForSecurity       -- ^ Spend everything from the address
+    | OptimizeForHighThroughput -- ^ No grouping, prefer confirmed addresses
```

Comment:
This is fine, as users are using `OptimizeForSecurity` by default and technically `OptimiseForHighThroughput` is a better policy for exchanges anyway.